docs/src/main/asciidoc/_admin-guide/subsystem-configuration/Elytron_OIDC_Client.adoc

```diff
@@ -404,6 +404,64 @@ your application's `web.xml` file, as shown in the example below:
 </web-app>
 ----
 
+== Logout Configuration
+
+OpenID Connect allows the client to configurate application based URIs that will
```

Comment:
Maybe this could be reworded to something like OpenID Connect provides the ability to log out users using RP-Initiated Logout, Back Channel Logout and Front Channel logout.

Reply:
done